tests/support/commands.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+import type { Devfile, FilterElem } from 'custom-types'
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      getDevfiles: typeof getDevfiles,
+      compareCardLength: typeof compareCardLength,
+      iterateThroughFilter: typeof iterateThroughFilter
+    }
+  }
+}
+
+type FilterElemsCallback = (selectedFilterElems: FilterElem[]) => void
+type DevfileCallback = (devfiles: Devfile[]) => void
+
+export const getDevfiles = (callback: DevfileCallback): void => {
+  cy.request('https://registry.devfile.io/index/all').then((response) => {
```

Comment:
IMO we should not make a call to the endpoint on tests. We should just have a mock_index.json and just parse that mock file for tests. We can do it later.